### PR TITLE
Fix minor issues in subscription settings

### PR DESCRIPTION
**Pull request by TomerAberbach**
Fixed a typo.

### CODE REVIEW

File: examples/source/interactivity-dynamic-content/Subscription_Settings/index.html [hunk 1]
```diff
@@ -19,7 +19,6 @@
     <meta charset="utf-8">
     <link rel="canonical" href="<% canonical %>">
     <title>Subscription Settings</title>
-    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
```
Comment (sebastianbenz) [inline examples/source/interactivity-dynamic-content/Subscription_Settings/index.html:22]: Should this go away?

Comment (TomerAberbach) [reply examples/source/interactivity-dynamic-content/Subscription_Settings/index.html:22]: Yeah, I think so. I noticed that the AMP playground automatically adds the `meta` viewport tag. You can open a snippet in the playground from the [page](https://amp.dev/documentation/examples/interactivity-dynamic-content/subscription_settings/?format=email) to see.

So if we don't remove this, then it'll be added twice

Comment (sebastianbenz) [reply examples/source/interactivity-dynamic-content/Subscription_Settings/index.html:22]: Looks like the playground doesn't correctly pre-process email snippets for the playground //cc @patrickkettner. Manually removing the viewport here should not be needed.

Comment (TomerAberbach) [reply examples/source/interactivity-dynamic-content/Subscription_Settings/index.html:22]: Ah I see. Let me revert this commit and I'll let @patrickkettner handle the viewport issue

**Comment by sebastianbenz** [comment]
Thanks!

**Pull request merged**